bot/src/test/kotlin/org/epilink/bot/AssetTest.kt

```diff
@@ -27,7 +27,7 @@ class AssetTest {
     @Test
     fun `Test load asset file`() {
         // Temporary file
-        val tmpFile = createTempFile("tmp_el_test")
+        val tmpFile = File.createTempFile("tmp_el_test", null)
```

Comment:
Why? `java.io` should be avoided like COVID-19

Reply:
Old function was marked as deprecated and File.createTempFile was explicitely written as one of the alternatives to use

Reply:
bruh, i'm sad